From c2cd5c07558b39b92e8f883e7df8f50de3c8ea86 Mon Sep 17 00:00:00 2001
From: Rob Herring <robh@kernel.org>
Date: Thu, 21 Nov 2019 16:57:03 -0600
Subject: [PATCH 1287/1852] dt-bindings: firmware: ixp4xx: Drop redundant
 minItems/maxItems

commit 7b11d505f6ca241b986fe76f628acfa9780d3f58 from
https://github.com/Xilinx/linux-xlnx.git

The minItems/maxItems default to the number of items in an 'items' list,
so drop the redundant specifying of them here.

Signed-off-by: Rob Herring <robh@kernel.org>
State: upstream (cb6192d647f79ae76128bce5216ea4f4e1776727)
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../firmware/intel,ixp4xx-network-processing-engine.yaml        | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/firmware/intel,ixp4xx-network-processing-engine.yaml b/Documentation/devicetree/bindings/firmware/intel,ixp4xx-network-processing-engine.yaml
index 8cb136c376fb..d144d4e258af 100644
--- a/Documentation/devicetree/bindings/firmware/intel,ixp4xx-network-processing-engine.yaml
+++ b/Documentation/devicetree/bindings/firmware/intel,ixp4xx-network-processing-engine.yaml
@@ -25,8 +25,6 @@ properties:
           - const: intel,ixp4xx-network-processing-engine
 
   reg:
-    minItems: 3
-    maxItems: 3
     items:
       - description: NPE0 register range
       - description: NPE1 register range
-- 
2.31.1

